nearcore/src/runtime/mod.rs

```diff
@@ -145,12 +146,35 @@ pub struct NightshadeRuntime {
 }
 
 impl NightshadeRuntime {
+    pub fn default(home_dir: &Path, store: Arc<Store>, genesis: &Genesis) -> Self {
+        Self::new(home_dir, store, genesis, vec![], false, None, None, RuntimeConfigStore::test())
+    }
+
+    pub fn new_with_config(
+        home_dir: &Path,
+        store: Arc<Store>,
+        config: &NearConfig,
+        trie_viewer_state_size_limit: Option<u64>,
```

Comment:
@Longarithm I had some questions here about `trie_viewer_state_size_limit` and `max_gas_burnt_view`, both of them are part of `config`, so I initially wanted this function to take these two values from `config` as well. But some call sites, almost all tests, specified these values separately. I am not too familiar with the code there and I thought you might have an idea why